src/Components/Publishing/Byline/__tests__/Date.test.tsx

```diff
@@ -37,11 +37,11 @@ describe("Date", () => {
 describe("#getDate", () => {
   const timestamp = "2017-02-22T19:22:05.709Z"
   const expectedFormattedDates = {
-    monthYear: "February 2017",
+    monthYear: "Feb 2017",
```

Comment:
Is this actually going to be changing date formats to shorthand? Is that something we expect to see replicated in the UI?

Reply:
Yes it's a recent change to our date formatting - https://www.notion.so/artsy/Formatting-Time-f6f4aaeeba414583a207013fec5a9794

Reply:
Ahh, I see. 👍